lib/getenv.js

```diff
@@ -69,7 +70,13 @@ var convert = {
       return (value === '1');
     }
   },
-  url: url.parse
+  url: url.parse,
+  date: function(value) {
+    if (dayjs(value).isValid()) {
```

Comment:
It would be great if we could find a way to do this without adding a comparably large dependency like dayjs (dayjs is much bigger than this library so it feels odd to add it as a first and only dependency).

Reply:
Yeah makes total sense. I'll look at how the `dayjs` library does the validation and see if its a simple replication or find some other way of validating!